src/NHibernate/Transform/AliasToBeanResultTransformer.cs

```diff
@@ -85,7 +85,7 @@ public override object TransformTuple(object[] tuple, String[] aliases)
 			{
 				result = _resultClass.IsClass
 							? _beanConstructor.Invoke(null)
-							: Cfg.Environment.ObjectsFactory.CreateInstance(_resultClass, true);
+							: Activator.CreateInstance(_resultClass, true);
```

Comment:
As discussed in #1758, this usage is not a dependency but a DTO instantiation. If the user wants some custom behavior for instantiating value types, he now would have to provide its own transformer.